README.md

```diff
@@ -51,66 +51,169 @@ directory to the web root of your server.
+language) directory's contents (with the exection of the [`en-CA.11tydata.js`](src/collections/pages/en-CA/en-CA.11tydata.js)
+file) up into `src/collections/pages`. If you do this, you'll also need to make the following change to the
+collection's directory data file, in this case [`src/collections/pages.11tydata.js`](src/collections/pages.11tydata.js):
+
+```diff
+- lang: data => EleventyI18nPlugin.LangUtils.getLanguageCodeFromInputPath(data.page.inputPath),
++ lang: data => data.defaultLanguage, // Use the default language for this collection
+```
+
+### Disabling Internationalization for the Entire Site
+
+If you need to disable internationalization for the entire site, you can do so by:
+
+1. Remove all but the default locale from the locales block of [`src/_data/config.json`](src/_data/config.json):
+
+   ```diff
+   "locales": [
+   -   "en-CA",
+   -   "fr-CA"
+   +   "en-CA"
+   ],
+   ```
+
+2. Delete the `editor` block's `i18n` section in [`src/admin/config.yml`](src/admin/config.yml):
+
+   ```diff
+   editor:
+     preview: false
+   - i18n:
+   -   structure: multiple_folders
+   -   locales: ["en-CA", "fr-CA"]
+   -   default_locale: "en-CA"
+   ```
+
+3. Deleting all directories but the default locale in the `/src/collections/<collection>/` directory. For example, to
+   disable localization for pages, delete the `/src/collections/pages/fr-CA/` directory.
+4. Modify the [`src/admin/config.yml`](src/admin/config.yml) collection block's `i18n` and `folder` properties for all
+   collections:
+
+   ```diff
+   - i18n: true
+   - folder: "src/collections/<collection>/"
+   + folder: "src/collections/<collection>/en-CA/" # Look for content to edit in the default locale directory.
+   ```
+
+   Or, remove all subdirectories of each collection directory, moving the `en-CA` (or other default language)
+   directory's contents (with the exection of the `en-CA.11tydata.js` file) up into `src/collections/<collection>`. If
+   you do this, you'll also need to make the following change to the collection's directory data file,
+   `src/collections/<collection>.11tydata.js`:
+
+   ```diff
+   - lang: data => EleventyI18nPlugin.LangUtils.getLanguageCodeFromInputPath(data.page.inputPath),
+   + lang: data => data.defaultLanguage, // Use the default language for this collection
+   ```
+
+   If you choose this option you won't need to change the `folder` configuration in [`src/admin/config.yml`](src/admin/config.yml).
+5. Modify your Eleventy configuration file to disable string translation in `eleventy-plugin-fluid`:
+
+   ```diff
+   eleventyConfig.addPlugin(fluidPlugin, {
+   +  i18n: false
+      ...
+   });
+   ```
+
+   Then make sure you remove any uses of the [`localizeData`](https://github.com/fluid-project/eleventy-plugin-fluid#localizedata)
+   helper or [`eleventy-plugin-i18n-gettext` functions](https://github.com/sgissinger/eleventy-plugin-i18n-gettext#api)
+   in your project. You can also delete `src/_locales`.
```

Comment:
Should probably mention the `src/_data/site.json` file which has localizations for the site. This data is used in conjunction with the gettext functions throughout.

Reply:
Data from `site.json` is passed as parameters but not as strings, e.g. [here](https://github.com/greatislander/trivet/blob/5f4a3774fe71065c2741ce561e76d6997ef7ce0c/src/_includes/layouts/base.njk#L12):

```njk
{% _ locale, "%s RSS feed", site[locale].name %}
```

Replacing the localization function wouldn't necessitate any changes to the `site.json` file; the data from it would just be used differently, like this:

```njk
{{ site[locale].name }} RSS feed
```

Reply:
An example of this should probably be included in the documentation. For example I didn't notice the `{% _ locale, ..., ... %}` short code mentioned in the `localizedData` and gettext plugin docs.